### DISCUSSION TITLE
Restore compact view in explorer address bar dropdown, lost since at least 22621.2861

### DISCUSSION THREAD

**Discussion by vicmich** [valinet/ExplorerPatcher#2678] [Ideas]
ExplorerPatcher version 22621.2861.62.2
Windows 11, OS Build 22621.2861

I'm hoping this within the scope of your project and that it can be done.  

I think I first started using the address bar dropdown during my six years with Windows 7, which ended five months ago. It was never perfect, in that it didn't share features with other Explorer navigation methods (typing directory name to jump to, right-clicking to open in a new tab/window, etc.)  But it did provide fast navigation in most daily use if you had an idea of where you were headed. 

As of Sunday, a Windows update has made it virtually unusable.   

I can't find any screenshot of the dropdown that I made prior to the update, but this link has detailed observations and screenshots that are similar: 
https://www.reddit.com/r/Windows11/comments/179bvhc/22h2_build_226212428_the_worst_windows_file/

**Big productivity issues for me:** 

1) There's now far too much vertical padding between directories in the dropdown. It's gone from a smaller font, single-line spacing to larger font, double-line spacing at least.  I'm wasting time scrolling down through dozens of directories. I'm going to get repetitive stress pain from the mouse wheel.

2) It no longer shows folder icons. For at least one key directory, I've customized the icon so I can immediately spot it and click.  No more.

3) The address bar itself has spaced itself out, and has changed behaviour in that it now (sometimes!) immediately shows paths that have been previously directly typed in a history dropdown.  As far as  I can remember, this only occurred before if you started typing, as a prompt. Now (sometimes!) it happens as soon as you drop the cursor in the bar, highlighting the existing path.

4) The one "upside" is that the right-arrow, greater-than caret (>), whatever it's called, seems to be back at the end of the path all the time if there are subdirs. Since I started on 11, it has been intermittent; I've often had to drop the cursor on the current dir before it appeared.

Anyway, if I could get back the previous, compact dropdown and the folder icons, I'd be happy enough. Eager to see if this might be possible.

**Comment by pyrates999** [comment]
Look at windhawk to see if you can add this back yourself using one of their custom mods or create your own mod.

You can try setting EP properties application and set file explorer **control interface** to use the **windows 7 command bar**.

**Comment by vicmich** [comment]
Thanks. I'd seen a reference to Windhawk somewhere, but a reply to that mentioned EP, which I already had, so thought I'd try here first. I'll give it a try.
